python/finufft/_interfaces.py

```diff
@@ -9,17 +9,24 @@
 # Anden 8/18/20: auto-made docstrings for the 9 simple/many routines
 
 
-import finufft._finufft as _finufft
 import numpy as np
 import warnings
 import numbers
 
-
-from finufft._finufft import default_opts
-from finufft._finufft import nufft_opts
-from finufft._finufft import finufft_plan
-from finufft._finufft import finufftf_plan
-
+from ctypes import byref
+from ctypes import c_longlong
+from ctypes import c_void_p
+
+from finufft._finufft import NufftOpts
+from finufft._finufft import _default_opts
```

Comment:
For single prec we also have finufftf_default_opts
This is currently no different, but it *might become so*, and in the docs we say for the single-prec user to use it.